cpp/oneapi/dal/test/archives.cpp

```diff
@@ -18,17 +18,24 @@
 #include "oneapi/dal/detail/array_utils.hpp"
 #include "oneapi/dal/detail/archives.hpp"
 #include "oneapi/dal/test/engine/common.hpp"
+#include "oneapi/dal/test/engine/fixtures.hpp"
 #include "oneapi/dal/test/engine/serialization.hpp"
 
 namespace oneapi::dal::test {
 
 namespace te = dal::test::engine;
 
-TEMPLATE_TEST("can write to binary_ouput_archive",
-              "[binary_ouput_archive]",
-              float,
-              double,
-              std::int32_t) {
+template <typename T>
+class archives_test : te::algo_fixture {
+public:
+};
```

Comment:
No logic in the class? Should algo_fixture just be used?

Reply:
YEs, its necessary just to add a class with template parameter for TEMPLATE_LIST_TEST_M macros usage

Reply:
Upd, it was replaced with more suitable macro